src/modules/navbar.js

```diff
@@ -0,0 +1,16 @@
+const navContainer = document.querySelector('.nav-container');
+
+const navigationMarkUp = `
+<ul class="nav-items">
+    <li class="nav-item"><a href="#" class="nav-link">link 1</a></li>
+    <li class="nav-item"><a href="#" class="nav-link">link 2</a></li>
+    <li class="nav-item"><a href="#" class="nav-link">link 3</a></li>
+    <li class="nav-item"><a href="#" class="nav-link">link 4</a></li>
+</ul>
+`;
+
+const renderNavigation = () => {
+  navContainer.insertAdjacentHTML('afterbegin', navigationMarkUp);
+};
+
+export default renderNavigation;
```

Comment:
Kindly delete this file as the header is static, we don't need to use Javascript for it